lib/src/core/configs/dialog_config.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:acme_theme_provider/src/core/component_config.dart';
+import 'package:flutter/material.dart';
+import 'package:json_theme/json_theme.dart';
+
+class DialogConfig extends ComponentConfig {
+  const DialogConfig({
+    required this.theme,
+    required this.insetPadding,
+    required this.clipBehavior,
+  });
+
+  final DialogTheme? theme;
+  final EdgeInsetsGeometry? insetPadding;
+  final Clip clipBehavior;
+
+  factory DialogConfig.fromMap(Map<String, dynamic> map) {
+    return DialogConfig(
+      theme: ThemeDecoder.decodeDialogTheme(map['theme']),
+      insetPadding: ThemeDecoder.decodeEdgeInsetsGeometry(map['insetPadding']),
```

Comment:
Is casting required ?

Reply:
Cannot be encoded or decode to edgeInsets type